nservicebus/sqlserver/deployment-options.md

```diff
@@ -0,0 +1,61 @@
+---
+title: SQL Server transport deployment options
+summary: Describes available SQL Server transport deployment options
+tags:
+- SQL Server
+- Transports
+redirects:
+- nservicebus/sqlserver/multiple-databases
+---
+
+In default configuration SQL Server transport assumes that all tables used for storing messages are located in a single catalog and within a single schema. The configuration can be changed to partition message storage between different schemas and catalogs. The decision can me made on per queue basis (including error and audit queues).
+
+The supported deployment options are:
```

Comment:
Should we separate this based on version? It's confusing to know what is and what isn't for what versions.